src/main/java/com/sele3/page/vietJet/SelectFlightPage.java

```diff
@@ -0,0 +1,197 @@
+            if (currentSize == previousSize) {
+                unchangedCount++;
+            } else {
+                unchangedCount = 0;
+                previousSize = currentSize;
+            }
+        }
+    }
+
+    @Step("Select the first cheapest ticket")
+    public void selectTheFirstCheapestTicket() {
+        scrollUntilAllPricesLoaded();
+        int minPrice = getCheapestPrice();
+        String formattedPrice = String.format("%,d", minPrice / 1000);
+        $$x(String.format("//p[contains(text(),'%s')]", formattedPrice)).get(0)
+                .scrollIntoView(false)
+                .shouldBe(Condition.visible)
+                .click();
+        clickContinueButton();
+    }
+
+    @Step("Click on Continue button")
+    public void clickContinueButton() {
+        getButtonByText((String) YamlUtils.getProperty("button.continue")).click();
+    }
+
+    public String getFromPlace() {
+        return $x(String.format(textByLabel, LabelType.FROM)).getText().split("\\(")[0].trim();
+    }
+
+    public String getToPlace() {
+        return $x(String.format(textByLabel, LabelType.TO)).getText().split("\\(")[0].trim();
+    }
+
+    public Passenger getPassengerInfo() {
+        int adults = getPassengerCount(LabelType.ADULT);
+        int children = getPassengerCount(LabelType.CHILDREN);
+        int infants = getPassengerCount(LabelType.INFANT);
+
+        return Passenger.builder()
+                .adultNumber(adults)
+                .childrenNumber(children)
+                .infantNumber(infants)
+                .build();
+    }
+
+    private int getPassengerCount(LabelType label) {
+        SelenideElement element = $x(String.format(passengerText, label.toString()));
+        if (element.exists()) {
+            String content = element.getText();
+            String regex = "(\\d+)\\s+" + label;
+            Matcher matcher = Pattern.compile(regex).matcher(content);
+            if (matcher.find()) {
+                return Integer.parseInt(matcher.group(1));
+            }
+        }
+        return 0; // Default to 0 if not found
+    }
+
+    public TicketData getTicketInfo(boolean isDepartureTicket) {
+        String formatter = (String) YamlUtils.getProperty("config.datetime_format");
+        String locale = (String) YamlUtils.getProperty("config.locale");
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern(formatter, new Locale(locale));
+
+        // Locate the section containing either "Departure flight" or "Return flight"
+        String flightDirection = isDepartureTicket
+                ? (String) YamlUtils.getProperty("label_Passenger_Page.departure_flight")
+                : (String) YamlUtils.getProperty("label_Passenger_Page.return_flight");
+        ElementsCollection section = $$x(String.format("//div[p[text()='%s']]/following-sibling::div//h5", flightDirection));
+
+        // Extract all <h5> texts within the selected section
+        List<String> h5Texts = section.texts();
```

Comment:
what is the size of h5Texts you're expected to get here? 3, 4 or 5?
If you definitely know, get by index per section, dont try to use loop to find it!